migration-guides/v8.0.0.md

```diff
@@ -0,0 +1,29 @@
+# v8.0.0 Migration Guide
+
+This version contains the following breaking changes:
+
+1. Upgraded Node version to 18.12.1
+2. Removed `node-sass` in favor of `sass`
+3. Upgraded webpack and related loaders
+
+Further explanation of each item is detailed below.
+
+---
+
+## 1. Upgraded Node version to 18.12.1
+
+Typically, the Node version of an application can be updated without any breaking change. However, Node 18 requires an upgraded version of `glibc`. This has impact on both Travis and the existing `node-sass` executable.
+
+For Travis, the default distribution used for builds and test execution does not have the version of `glibc` required by Node 18. To resolve this issue, a specific distribution is specified in the `.travis.yml` configuration file that has the correct `glibc` version.
```

Comment:
Can you reference the distribution name directly in this paragraph please? Any official docs from Travis to reference for more info?